### ISSUE TITLE
HTTP 500 Error on Configuration page

### ISSUE THREAD

**Issue by wangxu94** [splitbrain/dokuwiki#3215]
## Description

Since the latest upgrade, I have been getting a blank page with a HTTP 500 error when trying to navigate to the configuration page. I am still able to make changes to the configuration by editing the `local.php` file, and these changes do take effect.

## Specs:

Dokuwiki version: Release 2020-07-29 "Hogfather"
Skin: Bootstrap3 v2020-07-29; also tested with default Dokuwiki skin
Server OS: Ubuntu Server 16.04
Client OS: Windows 10
Browser: Firefox 79.0; also tested with Vivaldi 3.1.1929.48

## Console log:

```
GEThttps://<dokuwiki url>/doku.php?id=start&do=admin&page=config
[HTTP/1.0 500 Internal Server Error 29ms]

	
GET
	https://<dokuwiki url>/doku.php?id=start&do=admin&page=config
Status500
Internal Server Error
VersionHTTP/1.0
Transferred402 B (0 B size)

	
Cache-Control
	no-store, no-cache, must-revalidate
Connection
	close
Content-Length
	0
Content-Type
	text/html; charset=UTF-8
Date
	Sun, 02 Aug 2020 02:50:01 GMT
Expires
	Thu, 19 Nov 1981 08:52:00 GMT
Pragma
	no-cache
Server
	Apache/2.4.18 (Ubuntu)
Strict-Transport-Security
	max-age=31536000; includeSubDomains
Vary
	Cookie
X-Frame-Options
	SAMEORIGIN
	
Accept
	text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
Accept-Encoding
	gzip, deflate, br
Accept-Language
	en-US,en;q=0.5
Cache-Control
	max-age=0
Connection
	keep-alive
Cookie
	*****; DOKU_PREFS=list%23thumbs%23link%233%23align%231%23size%234%23ext_enabled%231%23ext_disabled%231%23ext_updatable%231%23wrapCtl%23soft%23plugin_prosemirror_useWYSIWYG%23; DokuWiki=****
	1
Host
	<dokuwiki url>
Referer
	https://<dokuwiki url>/doku.php?id=start&do=admin
Upgrade-Insecure-Requests
	1
User-Agent
	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0
Source map error: Error: NetworkError when attempting to fetch resource.
Resource URL: moz-extension://81378ce2-23b4-47a1-9aa7-c313f693a27d/browser-polyfill.min.js
Source Map URL: browser-polyfill.min.js.map

GEThttps://<dokuwiki url>/favicon.ico
[HTTP/1.1 404 Not Found 6ms]

	
GET
	https://<dokuwiki url>/favicon.ico
Status404
Not Found
VersionHTTP/1.1
Transferred590 B (280 B size)

	
Accept
	image/webp,*/*
Accept-Encoding
	gzip, deflate, br
Accept-Language
	en-US,en;q=0.5
Cache-Control
	max-age=0
Connection
	keep-alive
Cookie
	****; DOKU_PREFS=list%23thumbs%23link%233%23align%231%23size%234%23ext_enabled%231%23ext_disabled%231%23ext_updatable%231%23wrapCtl%23soft%23plugin_prosemirror_useWYSIWYG%23; DokuWiki=****
	1
Host
	<dokuwiki url>
User-Agent
	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0

```
​
Please let me know if you need any more information. Thanks!

**Comment by wangxu94** [comment]
Could not replicate this issue on a fresh install so obviously a user-error. My bad.

**Comment by Klap-in** [comment]
To discover the cause of a 500-error you have to check the webserver log to see where it stopped. See also https://www.dokuwiki.org/bugs#server_php_error_logs

**Comment by wangxu94** [comment]
@Klap-in Thanks for the tip — I did check the apache error log, but there didn't seem to be anything helpful there. I eventually discovered that the problem was caused by the [LaTeXit Plugin](https://www.dokuwiki.org/plugin:latexit).

**Comment by phy25** [comment]
@wangxu94 Thanks for reporting back the issue. We would appreciate it if you could also report this to https://github.com/Wrent/dokuwiki-plugin-latexit/issues.

**Comment by wangxu94** [comment]
@phy25 The latexit plugin doesn't seem to cause this issue on a fresh install. I also had other intersecting issues such as the editor and other buttons not loading correctly once I pointed a fresh install to my existing data directory. Clearing the cache did not help, and neither did running the cleanup script [here](https://www.dokuwiki.org/tips:maintenance). 

I eventually resolved the issue by copying the media and pages folders to a new data directory and using that. Unfortunately I don't have the time or patience to troubleshoot this any further so I'll just have to chalk this up to something in my existing data directory being borked. Thanks for your comment though, I appreciate your time and all your work on dokuwiki.
